packages/workflow-core/src/lib/logger.ts

```diff
@@ -0,0 +1,21 @@
+import { AnyRecord } from '@ballerine/common';
+
+export type LoggerInterface = {
+  log: (message: string, logData?: AnyRecord) => void;
+  error: (message: string, logData?: AnyRecord) => void;
+  warn: (message: string, logData?: AnyRecord) => void;
+  debug: (message: string, logData?: AnyRecord) => void;
+};
+
+/* eslint-disable no-console */
+export let logger: LoggerInterface = {
+  log: (message: string, logData: AnyRecord = {}) => console.log(message, logData),
+  error: (message: string, logData: AnyRecord = {}) => console.error(message, logData),
+  warn: (message: string, logData: AnyRecord = {}) => console.warn(message, logData),
+  debug: (message: string, logData: AnyRecord = {}) => console.debug(message, logData),
+};
```

Comment:
should be placed in common package

Reply:
Why?

Reply:
The logger itself can't be in the common, as it's a singleton that is specific to `workflow-core`. The only thing that can be placed in `common` is `LoggerInterface`, and I'm not sure it's worth it.